walkthroughs/writing-walkthroughs/walkthrough.adoc

```diff
@@ -443,7 +443,7 @@ Other attributes that are available to Walkthrough authors are:
 . Add another step to the numbered list in your Procedure:
 +
 ----
-. Click on link:\{ openshift-url }[Openshift Dashboard]
+. Click on link:\{openshift-url}[Openshift Dashboard]
```

Comment:
Just tested this out, it looks like we can remove the `\` here too and it still gives the expected result.

Reply:
yep,  that seems correct.
It's only when you set subs="attributes" that the url will be rendered, depending on the way asciidoctor.js is called (which is a different conversation)

Reply:
Ok, I'll remove the `\`